.github/workflows/ci-rocksdb-build.yml

```diff
@@ -17,6 +17,6 @@ jobs:
       - name: build rocksdb dependency
         run: bash ${GITHUB_WORKSPACE}/.github/scripts/install-rocksdb.sh
         env:
-          ROCKSDB_VERSION: v7.10.2
+          ROCKSDB_VERSION: v8.1.1
```

Comment:
https://github.com/Kava-Labs/infrastructure/blob/master/packer/kava-ami/kava-internal-testnet-rocksdb.pkrvars.hcl is what internal testnet is using, will need to update the config and update existing nodes using the kava-node-updater(see https://kava-labs.atlassian.net/wiki/spaces/ENG/pages/1105231874/In+Place+Updates+of+a+Kava+Network and https://kava-labs.atlassian.net/wiki/spaces/ENG/pages/1147174913/How+to+upgrade+Kava+Mainnet+or+Testnets#Kava-Node-Updater-CLI for details - for everywhere this change needs to be deployed to (e.g. https://github.com/Kava-Labs/infrastructure/blob/master/packer/kava-ami/kava-public-testnet-rocksdb.pkrvars.hcl and https://github.com/Kava-Labs/infrastructure/blob/master/packer/kava-ami/kava-14-rocksdb.pkrvars.hcl) - doesn't need to be done in this pr but @evgeniy-scherbina please add a shortcut ticket to the epic for this work to track it